README.md

```diff
@@ -0,0 +1,65 @@
+
+# Overview
+
+Terraform module for setting up infrastructure for [Shunting Yard](https://github.com/HotelsDotCom/shunting-yard).
+
+For more information please refer to the main [Apiary](https://github.com/ExpediaInc/apiary) project page.
+
+## Variables
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| aws_region | AWS region to use for resources. | string | - | yes |
+| cpu | The number of CPU units to reserve for the Shunting Yard container. Valid values can be 256, 512, 1024, 2048 and 4096. Reference: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html | string | `1024` | no |
+| memory | The amount of memory (in MiB) used to allocate for the Shunting Yard container. Valid values: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html | string | `4096` | no |
+| sy_ecs_task_count | Number of ECS tasks to create. | string | `1` | no |
```

Comment:
It doesn't support multiple copies at the same time at the moment. It will be something we will look to support in future but for now I have removed this option.